inc/options/pgcache.php

```diff
@@ -492,8 +738,64 @@
 			<tr>
 				<th>
 					<ul>
-						<li><?php _e( 'Enable <acronym title="Hypertext Transfer Protocol">HTTP</acronym> compression in the "<acronym title="Hypertext Markup Language">HTML</acronym>" section on <a href="admin.php?page=w3tc_browsercache">Browser Cache</a> Settings tab.', 'w3-total-cache' ); ?></li>
-						<li><?php _e( 'The <acronym title="Time to Live">TTL</acronym> of page cache files is set via the "Expires header lifetime" field in the "<acronym title="Hypertext Markup Language">HTML</acronym>" section on <a href="admin.php?page=w3tc_browsercache">Browser Cache</a> Settings tab.', 'w3-total-cache' ); ?></li>
+						<li>
+							<?php
+							echo wp_kses(
+								sprintf(
+									// translators: 1 opening HTML acronym tag, 2 closing HTML acronym tag,
+									// translators: 3 opening HTML acronym tag, 4 closing HTML acronym tag,
+									// translators: 5 opening HTML a tag to W3TC BrowserCache admin page, 6 closing HTML a tag.
+									__(
+										'Enable %1$sHTTP%2$s compression in the "%3$sHTML%4$s" section on %5$sBrowser Cache</a> Settings tab.',
+										'w3-total-cache'
+									),
+									'<acronym title="' . __( 'Hypertext Transfer Protocol', 'w3-total-cache' ) . '">',
+									'</acronym>',
+									'<acronym title="' . __( 'Hypertext Markup Language', 'w3-total-cache' ) . '">',
+									'</acronym>',
+									'<a href="admin.php?page=w3tc_browsercache">',
+									'</a>'
+								),
+								array(
+									'a'       => array(
+										'href' => array(),
+									),
+									'acronym' => array(
+										'title' => array(),
+									),
+								)
+							);
+							?>
+						</li>
+						<li>
+							<?php
+							echo wp_kses(
+								sprintf(
+									// translators: 1 opening HTML acronym tag, 2 closing HTML acronym tag,
+									// translators: 3 opening HTML acronym tag, 4 closing HTML acronym tag,
+									// translators: 1 opening HTML a tag to W3TC BrowserCache admin page, 2 closing HTML a tag.
+									__(
+										'The %1$sTTL%2$s of page cache files is set via the "Expires header lifetime" field in the "%3$sHTML%4$s" section on %5$sBrowser Cache%6$s Settings tab.',
+										'w3-total-cache'
+									),
+									'<acronym title="' . __( 'Time to Live', 'w3-total-cache' ) . '">',
+									'</acronym>',
+									'<acronym title="' . __( 'Hypertext Markup Language', 'w3-total-cache' ) . '">',
+									'</acronym>',
+									'<a href="admin.php?page=w3tc_browsercache">',
```

Comment:
Use `esc_url()` and `admin_url()` for the `href`.

Reply:
Added esc_url and admin_url to href